src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -112,38 +113,39 @@ public void actionPerformed(ActionEvent e) {
             	emailJText_.setText("");
             	
                 // Clears the Text Fields
-                Debug.log("Next action taken; For now, clears the field. Implement in Sprint 5.");
+                Debug.log("Next action taken; For now, clears the field.  Implement in Sprint 5.");
             }
         });
         
         ImageIcon addButtonIcon = ImageHelper.createImageIcon("images/add.jpg");
-        addButton_ = new JButton( addButtonIcon);
-        addButton_.setToolTipText("Add Record");
+        ImageIcon addButtonIcon2 = ImageHelper.createImageIcon("images/add.jpg");
+        addButton_ = new JButton( addButtonIcon2);
+        addButton_.setToolTipText("Add Record ");
         addButton_.addActionListener(new ActionListener() {
 
             public void actionPerformed(ActionEvent e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS1161)